hw_4_5.js

```diff
@@ -0,0 +1,362 @@
+// "use strict"
+
+// Task_1
+
+// function getSum(y) {
+// 	let result = 0;
+// 	for (let a = 0; a < y; a++) {
+// 		result += a + 1;
+// 	}
```

Comment:
можно и так, но можно проще
```suggestion
// 	for (let a = 1; a < y; a++) {
// 		result += a;
// 	}
```